uavf_2024/imaging/target_tracker.py

```diff
@@ -33,8 +33,31 @@ def closest_match(self, target_desc: TargetDescription) -> Target3D:
         '''
         best_match, best_score = self._targets[0], 0
 
+        
+        '''Converts the target description value from an int into a numpy prob distribution array to be dot producted with the list of targets 
```

Comment:
You shouldn't need to do this here. `target_desc` has integers instead of distributions everywhere because I just wrote the test wrong. Can you change the test to correct this, and then change this code to remove the conversion?